external-notification-service.openApi.yaml

```diff
@@ -0,0 +1,48 @@
+openapi: '3.0.1'
+info:
+  title: UKHO External Notification Service APIs
+  description: "This is a light weight open api spec for UKHO External Notification Service. One of its main purpose is to allow importing ENS APIs in APIM."
```

Comment:
I think we should clearly spell out that this spec is not to be used to be used for for API documentation. Basically don't use this as a full spec. Not sure the best way to  phrase this.

Reply:
I updated it to say: 
_This is a light weight open api spec for UKHO External Notification Service and it should not be used as formal API documentation. One of its main purpose is to allow importing ENS APIs in APIM._

Reply:
Looks good